streaming/index.js

```diff
@@ -621,6 +692,8 @@ const startServer = async () => {
         const delta = now - queued_at;
         const encodedPayload = typeof payload === 'object' ? JSON.stringify(payload) : payload;
 
+        streamingLatency.observe(delta);
```

Comment:
Apparently this is often `undefined`

Reply:
Looks like a lot of the events don't actually carry the `queued_at` property, and as such lag cannot be calculated.